repos.d/rpm/amazonlinux.yaml

```diff
@@ -0,0 +1,33 @@
+- name: amzn2
```

Comment:
I'd avoid unnecessary abbreviation and stick to how other repos are named, so probably `amazon_1` and `amazon_2`